mysql-test/mroonga/storage/function/highlight_html/t/open_tag_close_tag.test

```diff
@@ -0,0 +1,26 @@
+# Copyright(C) 2022 Horimoto Yasuhiro <horimoto@clear-code.com>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+
+--source ../../../../include/mroonga/not_embedded.inc
+--source ../../../../include/mroonga/have_mroonga.inc
+--source ../../../../include/mroonga/load_mroonga_functions.inc
+
+SELECT mroonga_highlight_html('Mroonga is the Groonga based storage engine.',
+                              'groonga',
+                              '<mark>' AS open_tag, '</mark>' AS close_tag) AS highlighted;
```

Comment:
How about using a valid HTML tag such as `<b>` and `</b>`?

Reply:
Could you also add the same text as `open_tag` and `close_tag` in the target text to confirm that `open_tag` and `close_tag` aren't used as keyword?

Reply:
`<mark>` seems to be a valid HTML tag.

https://www.w3schools.com/TAGS/tag_mark.asp

Reply:
> Could you also add the same text as `open_tag` and `close_tag` in the target text to confirm that `open_tag` and `close_tag` aren't used as keyword?

The test for that cases was already added.
Please confirm it.

https://github.com/mroonga/mroonga/blob/8dd904870fe721804dde8dca4ce719f9d8edcf49/mysql-test/mroonga/storage/function/highlight_html/t/open_tag_close_tag_not_added_to_keywords.test

[mysql-test/mroonga/storage/function/highlight_html/r/open_tag_close_tag_missing_close_tag.result](https://github.com/mroonga/mroonga/blob/8dd904870fe721804dde8dca4ce719f9d8edcf49/mysql-test/mroonga/storage/function/highlight_html/r/open_tag_close_tag_missing_close_tag.result)

Reply:
> `<mark>` seems to be a valid HTML tag.

Wow! I didn't know that!
We should have used it for the default tag...

Reply:
> > Could you also add the same text as `open_tag` and `close_tag` in the target text to confirm that `open_tag` and `close_tag` aren't used as keyword?
> 
> The test for that case was already added. Please confirm it.
> 
> https://github.com/mroonga/mroonga/blob/8dd904870fe721804dde8dca4ce719f9d8edcf49/mysql-test/mroonga/storage/function/highlight_html/t/open_tag_close_tag_not_added_to_keywords.test

Could you use different text such as `[` not `open_tag` to ensure `open_tag` argument is used?

```sql
SELECT mroonga_highlight_html('Groonga should be highlighted but [ and ] should not be highlighted',
                              'Groonga', '[' AS open_tag, ']' AS close_tag) AS highlighted;
```